### PR TITLE
Prettify match/cosmetics, globalize tooltip style

### PR DESCRIPTION
**Pull request by blukai**

### CODE REVIEW

**Comment by blukai** [comment]
image

**Comment by blukai** [comment]
In #336 I fix spinner's err but in my next pr bring it back. My bad, fix it again. 🍀

File: src/components/Match/matchColumns.jsx [hunk 1]
```diff
@@ -486,16 +493,57 @@ export const runesColumns = [heroTdColumn]
...
+  legendary: '#D32CE6',
+  immortal: '#E4AE33',
+  arcana: '#ADE55C',
+  ancient: '#EB4B4B',
+};
+export const cosmeticsColumns = [{
+  displayName: 'Player',
+  field: 'player_slot',
+  cosmetics: true,
+  displayFn: (row, col, field, i) => heroTd(row, col, field, i, false, 'side'),
```
Comment (howardchung) [inline src/components/Match/matchColumns.jsx]: why side instead of MMR?

File: src/components/Match/matchPages.jsx [hunk 2]
```diff
@@ -232,9 +232,12 @@ const matchPages = [{
 }, {
   name: strings.tab_cosmetics,
   parsed: true,
-  content: match => (<div>
-    <TeamTable match={match} columns={cosmeticsColumns} heading={strings.heading_cosmetics} />
-  </div>),
+  content: match => (
+    <div>
+      <Heading title={strings.heading_cosmetics} />
```
Comment (howardchung) [inline src/components/Match/matchPages.jsx]: shouldn't we be consistent and use split tables everywhere if we are going to use them at all?  Seems strange to break from that just for cosmetics.

File: src/components/Match/matchColumns.jsx [hunk 3]
```diff
@@ -486,16 +493,57 @@ export const runesColumns = [heroTdColumn]
...
+  sortFn: true,
+}, {
   displayName: strings.th_cosmetics,
   field: 'cosmetics',
+  cosmetics: true,
```
Comment (blukai) [inline src/components/Match/matchColumns.jsx]: To hide rows in cosmetics table for players that have no cosmetics.

File: src/components/Match/matchColumns.jsx (hunk 1 above)
Comment (blukai) [inline src/components/Match/matchColumns.jsx]: Nvm, I have another better idea, I'll revert this change

File: src/components/Match/matchPages.jsx (hunk 2 above)
Comment (blukai) [inline src/components/Match/matchPages.jsx]: Actually cosmetics IMO shouldn't be splitted for teams as other tables. It's not a team info, its personal.

File: src/components/Table/Table.jsx [hunk 4]
```diff
@@ -37,6 +37,9 @@ const getTable = (data, columns, sortState, sortField, sortClick) => (
           <MaterialTableRow
             key={index}
             className={styles.row}
+            style={{
```
Comment (howardchung) [inline src/components/Table/Table.jsx]: I think it's fine to keep empty rows.  Alternatively run the input data through a filter function?

**Comment by blukai** [comment]
@howardchung I guess we should flow cosmetics images through https like for other dota images.

**Comment by howardchung** [comment]
We could, although I don't know if it's needed right away.  Open an issue?

**Pull request merged**